backend/app/config.py

```diff
@@ -15,6 +15,12 @@
 RUN_OUTPUT_FOLDER = "output"
 REPOSITORY_NAME = "rail-exp-templates"
 TEMP_DIRNAME = "temp"
+RESERVED_ENV_VARS = [
```

Comment:
Maybe turn this into Enum, so we can rely on it in different parts in the code. Currently it is hardcoded here and in one more place